apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.js

```diff
@@ -178,6 +185,25 @@ router.post('/', function (req, res, next) {
         }
       });
     });
+  } else if (req.body.__action === 'unsafe_sharing_set_add') {
+    debug('Add question to sharing set');
+    const questionSharingEnabled = features.enabledFromLocals('question-sharing', res.locals);
```

Comment:
```suggestion
    const questionSharingEnabled = await features.enabledFromLocals('question-sharing', res.locals);
```

(otherwise `questionSharingEnabled` will just be a `Promise` and it will always be truthy).

Reply:
ugh, this footgun is definitely my least favorite part about async/await. Thanks.

Reply:
I ended up just using `.then()...` instead to avoid having to refactor this large function to be async.